### PR TITLE
fix the link of the online form

### PR DESCRIPTION
**Pull request by zhyuri**

### CODE REVIEW

**Comment by zhyuri** [comment]
Thanks so much for your videos on the Youtube, I really learned a lot from it. Please keep on doing this.

File: README.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ series [#justforfunc](http://youtube.com/c/justforfunc).
 
 ### Have an idea?
 
-Submit your ideas [here](https://form.justforfunc.om) or directly
+Submit your ideas [here](https://form.justforfunc.com) or directly
```
Comment (akshaychhajed) [inline README.md]: @zhyuri Looks like https does not work. Please make it http link

Comment (zhyuri) [inline README.md]: oh, you are right. I will fix it.

**Comment by campoy** [comment]
Thanks for the fix!

**Pull request merged**